guardrails/classes/rc.py

```diff
@@ -0,0 +1,71 @@
+import logging
+import os
+from dataclasses import dataclass
+from os.path import expanduser
+from typing import Optional
+
+from guardrails.classes.generic.serializeable import Serializeable
+from guardrails.utils.casting_utils import to_bool
+
+BOOL_CONFIGS = set(["no_metrics", "enable_metrics", "use_remote_inferencing"])
```

Comment:
not related to this pr but for my education whats the difference between no metrics and enable metrics?

Reply:
`no_metrics` is the old config settings (deprecated), `enable_metrics` is the current settings.  The practical difference was changing it from opt-out to opt-in